dataPrepII_PCC/makePCCRateFile.py

```diff
@@ -109,13 +104,13 @@ def addScanInfoToTree(chain, inData, intermediateFile):
 
 
 
-def fillClusterCounts(pcc_tree, tree_scan_info, inData, PCC_BCID):
+def fillCounts(pcc_tree, tree_scan_info, inData, PCC_BCID, DataType):
```

Comment:
If the suggestion above concerning rates per layer is ok, add

PixelLayer as an argument